homework/lesson 6/src/ru/otus/java/homework/HomeworkTemplate.java

```diff
@@ -0,0 +1,56 @@
+package ru.otus.java.homework;
+
+import java.util.Scanner;
+
+/**
+ * Заготовка для выполнения ДЗ "Система тестирования".
+ */
+public class HomeworkTemplate {
+    public static void main(String[] args) {
+        // Переменные для хранения количества правильных и неправильных ответов
+        int correctCount = 0, wrongCount = 0;
+        final String correct = "Правильно";
+        final String incorrect = "Неправильно";
+        final String err_answer = "Такого варианта ответа не существует";
+
+        String[] questions = {"сколько дней в неделе?", "можно ли пройти ревью с первого раза?"};
+
+        String[][]answerOptions = {{"6","7","8","9"},{"да", "нет"}};
+
+        int [] correctAnswers = {2, 2};
+
+        Scanner scanner = new Scanner(System.in);
+        String v_answer;
+        for (int i = 0; i < questions.length; i++) {
+
+            System.out.println("Вопрос:" + questions[i]);
+
+            for(int i1 = 0; i1 < answerOptions[i].length; i1++)
```

Comment:
```suggestion
            for(int j = 0; j < answerOptions[i].length; j++)
```
Переменные для вложенных циклов часто называют `i`, `j`, `k`, ...

Reply:
done